src/Context/UserContext.php

```diff
@@ -9,6 +9,8 @@
  */
 class UserContext extends RawWordpressContext
 {
+    use Awareness\UserAwareContextTrait, Awareness\CacheAwareContextTrait;
```

Comment:
UserContext does not appear to need CacheAwareContextTrait

Reply:
yeah - there are a couple of situations like that. There's also a few things not in the traits which should be. I'm going to do an 'audit' this evening and check all is as it should be :)